File: lint/src/main/java/net/grandcentrix/thirtyinch/IssueRegistry.java
package net.grandcentrix.thirtyinch;

import com.android.tools.lint.detector.api.Issue;

import java.util.ArrayList;
import java.util.List;

public class IssueRegistry extends com.android.tools.lint.client.api.IssueRegistry {

    @Override
    public List<Issue> getIssues() {
        List<Issue> issues = new ArrayList<>();
        issues.add(MissingTiViewImplementationDetector.ISSUE);
        return issues;
    }
}

File: lint/src/main/kotlin/net/grandcentrix/thirtyinch/IssueRegistry.kt
package net.grandcentrix.thirtyinch

class IssueRegistry : com.android.tools.lint.client.api.IssueRegistry() {

    override fun getIssues() = listOf(
            MissingViewInThirtyInchDetector.ISSUE,
            MissingViewInCompositeDetector.ISSUE
    )
}
